spec/requests/v1/padlock/authentication_requests_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe V1::Padlock::AuthenticationController, type: :request do
+  describe 'authentication' do
```

Comment:
use name of action here, please, similar way you've done that in `ArticlesController` spec